command/sub8_missions/nodes/move_command

```diff
@@ -95,6 +146,8 @@ if __name__ == '__main__':
         help="Make end goal have zero roll and pitch.")
     parser.add_argument('-b', '--blind', action='store_true',
         help="Do not check if waypoint is safe.")
+    parser.add_argument('-gt', '--gteleport',metavar='object', type=str,
+        help="Teleport sub in Gazebo to object. Does nothing if not in Gazebo. Takes a string meaning quotes required around input if it is not a shortcut implemented through yaml.")
```

Comment:
A couple of points although I don't think this should be implemented as an option:
* Metavars are usually short and capitalized
* Object isn't a very descriptive name. I would go for something like 'checkpoint'
* You usually want help messages to be a bit more concise